misc/docs/source/tutorial/code_snippets/trigger_tutorial.rst

```diff
@@ -396,6 +397,27 @@ For :func:`~obspy.signal.trigger.ar_pick`, input and output are in seconds.
 This gives the output 30.6350002289 and 31.2800006866, meaning that a P pick at
 30.64s and an S pick at 31.28s were identified.
 
+AIC - Akaike Information Criterion
+===========
```

Comment:
The line of `=` signs should match the header line

Reply:
done